scripts/check_duplicates.py

```diff
@@ -78,7 +78,9 @@ def check_duplicates(sources, os_name, os_codename):
     print("checking duplicates")
     db_name_view = {}
     has_duplicates = False
-    view = lookup.get_rosdep_view(DEFAULT_VIEW_KEY, verbose=None)  # to call init
+    # to avoid merge views
+    #view = lookup.get_rosdep_view(DEFAULT_VIEW_KEY, verbose=None)  # to call init
```

Comment:
can this line be removed? or do we want to keep it for bookkeeping? if we want to keep it can you please use `# ` (with a space after) to comply with pep8?